agent/consul/prepared_query_endpoint_test.go

```diff
@@ -77,7 +78,7 @@ func TestPreparedQuery_Apply(t *testing.T) {
 	query.Query.Service.Failover.NearestN = 0
 	query.Query.Session = "nope"
 	err = msgpackrpc.CallWithCodec(codec, "PreparedQuery.Apply", &query, &reply)
-	if err == nil || !strings.Contains(err.Error(), "failed session lookup") {
+	if err == nil || !strings.Contains(err.Error(), "invalid session") {
```

Comment:
Looks like this string change is causing tests to fail:
```
prepared_query_endpoint_test.go:82: bad: failed session lookup: index error: UUID must be 36 characters
```

Reply:
@freddy Would this be fixed by the oss modifications from your sessions namespacing PR.

Reply:
Good catch! I think I took it from oss, but didn't consider it might have not made its way into ent. But thats entirely possible.

Reply:
ok, I am putting it back. it should be fixed by an oss merge.

Reply:
Yep that should be fixed when I make the OSS PR.